app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -71,6 +71,7 @@ private fun ConversationScreen(
     onSendButtonClicked: () -> Unit,
     onSendAttachment: (AttachmentBundle?) -> Unit,
     onDownloadAsset: (String) -> Unit,
+    onImageFullMode: (String) -> Unit,
```

Comment:
What do you think of having a different name for this argument ? Something like `onImageFullScreenMode` ? Maybe it's only me, but I think "Full" needs like a companion lastname :P